src/main/java/io/confluent/connect/jdbc/source/JdbcSourceConnectorConfig.java

```diff
@@ -36,8 +37,16 @@
 public class JdbcSourceConnectorConfig extends AbstractConfig {
 
   public static final String CONNECTION_URL_CONFIG = "connection.url";
-  private static final String CONNECTION_URL_DOC = "JDBC connection URL for the database to load.";
-  private static final String CONNECTION_URL_DISPLAY = "Connection Url";
+  private static final String CONNECTION_URL_DOC = "JDBC connection URL.";
```

Comment:
Mentioning "Optionally including username and password, or use connection.user and connection.password"? Or is this unnecessary given other fields will be visible?